lib/src/runner/configuration/args.dart

```diff
@@ -92,17 +93,18 @@ final ArgParser _parser = (() {
           'debuggability.',
       defaultsTo: true);
 
+  Map reporterDescriptions = {};
```

Comment:
Put types on the right when possible.

`var reporterDescriptions = <String, String>{};`